catroid/src/org/catrobat/catroid/web/ServerCalls.java

```diff
@@ -82,102 +103,162 @@
 
 	public static boolean useTestUrl = false;
 	private String resultString;
-	private ConnectionWrapper connection;
 	private String emailForUiTests;
-	private int uploadStatusCode;
+	private final OkHttpClient okHttpClient;
+	private final Gson gson;
 
 	private ServerCalls() {
-		connection = new ConnectionWrapper();
+		okHttpClient = new OkHttpClient();
+		okHttpClient.setConnectionSpecs(Arrays.asList(ConnectionSpec.MODERN_TLS));
```

Comment:
This means that cleartext connections are not allowed. You'd have to add `ConnectionSpec.CLEARTEXT` if you want that.